HIFIMAGS/HIFIMAGS-IOC-01App/Db/hifimags.db

```diff
@@ -0,0 +1,239 @@
+### Generic Records
+
+record(bo, "$(P)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM)")
+    info("testparam","testvalue")
+}
+
+record(bo, "$(P)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+### Operation Mode
+
+record(mbbo, "$(P)OPMODE:SP") {
+  field(DESC, "Operation Mode")
+  field(DTYP, "Soft Channel")
+  field(SCAN, "1 second")
```

Comment:
Because I missed removing the scanning when I started making those corrections as I wasn't thinking when I created.

Good point re archiving and interest on the OPMODE, I've added those fields